src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -14,11 +14,11 @@
 	$: heroContent = usdTotal || summary;
 </script>
 
-<div class="pt-6 space-y-10">
+<div class="pt-6">
 	<Header back={back === 'header'} />
 
 	<article
-		class="flex flex-col rounded-lg pb-6 relative main 2xl:mt-[-70px] items-center"
+		class="flex flex-col rounded-lg pt-10 pb-6 relative main 2xl:mt-[-70px] items-center"
```

Comment:
Have you double check that modifying this does not lead to any conflicts with everything single other modals and overlays used in the app?

Reply:
yes yes, checked a few modals; but even because I introduced that `space-y-10`  and removed `pt-` from `article`, in PR #2455 , and it was all good before

Reply:
ok